SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -92,6 +94,11 @@ class CVariable {
                                                        note that this variable cannnot be static, it is possible to
                                                        have different number of nVar in the same problem. */
   su2double *Solution_Adj_Old;    /*!< \brief Solution of the problem in the previous AD-BGS iteration. */
+  
+  int *Input_AdjIndices,          /*!< \brief Indices of Solution variables in the adjoint vector. */
```

Comment:
Just for curiosity, we usually use unsigned long for large indices, is int the native type of CoDi? if so how large of a tape does that support per mpi rank?

Reply:
I think it is but I'll make sure.